Add wxPU_CONTAINS_CONTROLS style for wxPopupWindow

NPDevelopmentCode opened the pull request:
This restores the default behavior of a popup window in MSW to the
behavior it had before 56c4191. The new flag added by this commit can
be used to give the popup window the behavior from after that commit.

The old behavior kept the popup from taking focus from its parent window
but left some controls unworking. The new behavior has the popup take
focus and lets all controls work.

NewPagodi commented:
The one thing I didn't restore was that wxPopupWindow used to add the style wxPOPUP_WINDOW in its Create method. But since this flag doesn't seem to be used anywhere (it appears only in defs.h, the dfb sources, and the carbon sources), I think its unnecessary.

Commit:
Add wxPU_CONTAINS_CONTROLS style for wxPopupWindow

This restores the default behavior of a popup window in MSW to the
behavior it had before 56c4191. The new flag added by this commit can be
used to give the popup window the behavior from after that commit.

The old behavior kept the popup from taking focus from its parent window
but left some controls unworking. The new behavior has the popup take
focus and lets all controls work.

NewPagodi commented:
Since in MSW wxPopupTransientWindow now works by responding to loss of activation and the old style of popup doesn't activate in the first place, wxPopupTransientWindow needs to have the new flag so that it will use the new popup style.  I've made a small update to to add a Create method which simply calls the base class Create method with the new flag added.

vadz commented:
No comments, thanks! I'll merge this when the CI build finishes.
